### ISSUE TITLE
Setting "Overhang threshold" to any nonzero value creates undesired interface pattern artifacts

### ISSUE THREAD

**Issue by Gatulibu** [alexrj/Slic3r#3895]
_Expected Result_:
The topmost interface pattern is rectilinear:

![img_1-2-9](https://cloud.githubusercontent.com/assets/1441713/25212227/77448e78-2589-11e7-9d2d-e5f18f90e39a.JPG)
_Version used 1.2.9 stable (W7-x64), all settings default + support enabled._

_Actual Result_ with "Overhang threshold" set to 25:
The topmost interface layer seems to have some hexagonal lines/artifacts at the outer part of the support:

![img_1-2-9-ot25](https://cloud.githubusercontent.com/assets/1441713/25212234/7db7f1f0-2589-11e7-8690-a3063449876b.JPG)
_Version used 1.2.9 stable (W7-x64), all settings default + support enabled + Overhang threshold set to 25°._

The new % setting introduced in the dev versions seems to create the undesired pattern with every value I tried.
![img_1-3-0-2017 04 19 958 3274b75](https://cloud.githubusercontent.com/assets/1441713/25212243/862cc0d6-2589-11e7-84bc-827d86f8a7fc.JPG)
_Version used dev (2017.04.19.958.3274b75 W7-x64), all settings default + support enabled._

Test STL:
supporttest.stl
[supporttest.zip](https://github.com/alexrj/Slic3r/files/939911/supporttest.zip)

**Comment by lordofhyphens** [comment]
As a FYI: 200% is equivalent to the old value of 0 (or auto).

**Comment by foreachthing** [comment]
> The topmost interface layer seems to have some hexagonal lines/artifacts at the outer part of the support

I agree ... to a point: I, too, think there is _too much support_ (in X and Y) and no way to control that. I don't think of those jagged lines as artifacts as much as unnecessary. Especially outside the part area to support - where it's only wasted plastic.
The user should be able to control the offset (X and Y) of the support material (as one can in Cura).

**Comment by lordofhyphens** [comment]
There is a PR for that, actually. It is pretty hacky though.
